generators/add-web-assets/exc-react/templates/src/components/About.js

```diff
@@ -0,0 +1,54 @@
+/*<% if (false) { %>
+Copyright 2019 Adobe. All rights reserved.
```

Comment:
2020?

Reply:
Should we make this another PR, 
There are multiple files, which needs this update

Reply:
I think new files get a 2020 copyright but not existing files